app/services/docker/pluginService.js

```diff
@@ -60,5 +60,17 @@ angular.module('portainer.services')
     return servicePlugins(systemOnly, 'Network', 'docker.networkdriver/1.0');
   };
 
+  service.loggingPlugins = function(systemOnly) {
+    var deferred = $q.defer();
```

Comment:
You should have kept `return servicePlugins(systemOnly, 'Log', 'docker.logdriver/1.0');` in the function `loggingPlugins`, that was correct.

Reply:
Aw! Sorry for so many trials and error, I'm not an Angular guy :) but I'll do my bes.

Reply:
No worries, you're actually doing pretty well !